cscs-checks/libraries/math/trilinos_compile_run.py

```diff
@@ -49,3 +51,13 @@ def __init__(self, linkage):
     def set_cxxflags(self):
         flags = self.prgenv_flags[self.current_environ.name]
         self.build_system.cxxflags = flags
+
+    @rfm.run_before('compile')
+    def set_prebuild_cmd(self):
+        if self.current_environ.name.startswith('PrgEnv-intel'):
+            if '20.06' in os.getenv('MODULERCFILE', ''):
+                self.modules += ['cray-netcdf-hdf5parallel']
+                self.prebuild_cmds = [
+                    'ln -s $CRAY_NETCDF_HDF5PARALLEL_PREFIX/lib/pkgconfig/'\
+                        'netcdf-cxx4_parallel.pc netcdf_c++4_parallel.pc',
+                    'export PKG_CONFIG_PATH=`pwd`:$PKG_CONFIG_PATH']
```

Comment:
You should set the `variables` instead of using the `export` here. Also, since ReFrame runs always from the stage directory, you can simply export `PKG_CONFIG_PATH=.:$PKG_CONFIG_PATH`.

Reply:
Done.